README.md

```diff
@@ -2,15 +2,20 @@
 
 The Samvera CircleCI Orb is meant to make testing Samvera and Samvera-based projects easier.
 
-Code: [![Build Status](https://circleci.com/gh/samvera/samvera-circleci-orb.svg?style=svg)](https://circleci.com/gh/samvera/samvera-circleci-orb)
+Code: [![Samvera Core Component](https://img.shields.io/badge/samvera-core--component-brightgreen)](https://github.com/samvera/maintenance#samvera-core-components)
+[![Build Status](https://circleci.com/gh/samvera/samvera-circleci-orb.svg?style=svg)](https://circleci.com/gh/samvera/samvera-circleci-orb)
+[![Version](https://img.shields.io/badge/orb-1.0.3-blue)](https://circleci.com/developer/orbs/orb/samvera/circleci-orb?version=1.0.3)
```

Comment:
It looks like this would have to be manually updated with each release and it looks like circleci provides a badge service.  So maybe this could become:
```suggestion
[![Version](https://badges.circleci.com/orbs/samvera/circleci-orb.svg)](https://circleci.com/developer/orbs/orb/samvera/circleci-orb)
```